### PR TITLE
aws: support firehose

### PR DESCRIPTION
**Pull request by okazu-dm**
Hello.
This change support "aws_kinesis_firehose_delivery_stream" like following command.

```
terraformer import aws  -r firehose --regions "ap-northeast-1"
```

FYI
https://www.terraform.io/docs/providers/aws/r/kinesis_firehose_delivery_stream.html
https://aws.amazon.com/kinesis/data-firehose/

Thank you.

### CODE REVIEW

File: providers/aws/firehose.go [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+type FirehoseGenerator struct {
+	AWSService
+}
+
+func (g FirehoseGenerator) createResources(sess *session.Session, streamNames []*string, region string) []terraform_utils.Resource {
+
```
Comment (sergeylanzman) [inline providers/aws/firehose.go]: Empty line - delete

File: providers/aws/firehose.go [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+import (
+	"github.com/GoogleCloudPlatform/terraformer/terraform_utils"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/firehose"
+)
+
+// Copyright 2018 The Terraformer Authors.
```
Comment (sergeylanzman) [inline providers/aws/firehose.go]: License first in file

File: providers/aws/firehose.go [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+type FirehoseGenerator struct {
+	AWSService
+}
+
+func (g FirehoseGenerator) createResources(sess *session.Session, streamNames []*string, region string) []terraform_utils.Resource {
```
Comment (sergeylanzman) [inline providers/aws/firehose.go]: You need region here?

Comment (okazu-dm) [reply providers/aws/firehose.go]: Oops, I'll remove that argument.

File: providers/aws/firehose.go (hunk 1 above)
Comment (okazu-dm) [reply providers/aws/firehose.go]: deleted

File: providers/aws/firehose.go (hunk 2 above)
Comment (okazu-dm) [reply providers/aws/firehose.go]: Moved that.

**Comment by okazu-dm** [comment]
Thanks for your review! I finished to add README and fix codes.

**Pull request merged**